### DISCUSSION TITLE
how to complete LFX mentorship task few steps

### DISCUSSION THREAD

**Discussion by Aman123lug** [cncf/mentoring#831] [Q&A]
how to complete LFX mentorship task explain some step to approch ?

**Comment by johurul000** [comment]
I guess there might be difference among the task assigned by different participating organizations.

**Reply by johurul000** [reply]
And I think resume, Cover letter etc will be there.

**Reply by Aman123lug** [reply]
yes i selected a project and i submited my resume and my cover letter. but i want to complete assignment and is there any time limit to complete task
